instrumentation/runtime-metrics/runtime-metrics-java17/javaagent/src/main/java/io/opentelemetry/instrumentation/javaagent/runtimemetricsjava17/Java17RuntimeMetricsInstaller.java

```diff
@@ -3,38 +3,41 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.instrumentation.javaagent.runtimetelemetryjfr;
+package io.opentelemetry.instrumentation.javaagent.runtimemetricsjava17;
 
 import com.google.auto.service.AutoService;
 import io.opentelemetry.api.GlobalOpenTelemetry;
 import io.opentelemetry.api.OpenTelemetry;
-import io.opentelemetry.instrumentation.runtimetelemetryjfr.JfrTelemetry;
+import io.opentelemetry.instrumentation.runtimemetricsjava17.RuntimeMetrics;
 import io.opentelemetry.javaagent.extension.AgentListener;
 import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk;
 import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
 
 /** An {@link AgentListener} that enables runtime metrics during agent startup. */
 @AutoService(AgentListener.class)
-public class RuntimeMetricsInstallerJfr implements AgentListener {
+public class Java17RuntimeMetricsInstaller implements AgentListener {
 
   @Override
   public void afterAgent(AutoConfiguredOpenTelemetrySdk autoConfiguredSdk) {
     ConfigProperties config = autoConfiguredSdk.getConfig();
 
     OpenTelemetry openTelemetry = GlobalOpenTelemetry.get();
-    JfrTelemetry jfrTelemetry = null;
+    RuntimeMetrics runtimeMetrics = null;
     /*
     By default don't use any JFR metrics. May change this once semantic conventions are updated.
-    If enabled, default to only the metrics not already covered by runtime-telemetry-jmx
+    If enabled, default to only the metrics not already covered by runtime-metrics-java8
     */
 
-    if (config.getBoolean("otel.instrumentation.runtime-telemetry-jfr.enable-all", false)) {
-      jfrTelemetry = JfrTelemetry.builder(openTelemetry).enableAllFeatures().build();
-    } else if (config.getBoolean("otel.instrumentation.runtime-telemetry-jfr.enabled", false)) {
-      jfrTelemetry = JfrTelemetry.create(openTelemetry);
+    if (config.getBoolean("otel.instrumentation.runtime-metrics-java17.enable-all", false)) {
+      runtimeMetrics = RuntimeMetrics.builder(openTelemetry).enableAllFeatures().build();
+    } else if (config.getBoolean("otel.instrumentation.runtime-metrics-java17.enabled", false)) {
+      runtimeMetrics = RuntimeMetrics.create(openTelemetry);
```

Comment:
I wonder if we should enable both JMX+JFR default metrics if `otel.instrumentation.runtime-metrics.enabled` is true.

Reply:
Hmm yeah we talked a bit about this [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7886#discussion_r1123453779). It was decided that we'd wait for the semantic conventions to be updated first in-case the JFR metrics end up changing.

Reply:
Hmm, I see.
At the very least, let's add a condition checking `otel.instrumentation.runtime-metrics.enabled` somewhere in this method -- the current version of that is impossible to turn off on java 17+

Reply:
Yes that's a good point. I'll add the same check that's done in `Java8RuntimeMetricsInstaller` before enabling the JMX gathered metrics.